src/index.html

```diff
@@ -5,8 +5,332 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="header__container">
+        <a href="index.html" class="header__company-logo">
+          <img src="./images/logo.png" alt="logo">
+        </a>
+      </div>
+
+      <nav class="header__navigation">
+        <ul class="header__menu">
+          <li class="header__menu-item">
+            <a href="#" class="header__menu-link is-active">Apple</a>
```

Comment:
IMHO, is a little better because we can see the content without reading the whole line

```suggestion
            <a href="#" class="header__menu-link is-active">
              Apple
            </a>
```

Reply:
Done